src/hooks/screen/useChatInfoHook.ts

```diff
@@ -24,15 +33,10 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
     selectedUser,
     blockModalRef,
     isLoadingInitChat
-  } = useGroupInfo();
-  const {localDb} = useLocalDatabaseHook();
-  const [loadingChannelInfo, setLoadingChannelInfo] = React.useState<boolean>(false);
-  const {isLoadingFetchingChannelDetail, selectedChannel, goBack} = useChatUtilsHook();
-  const [myUserId] = React.useContext(Context).profile;
-  const navigation = useNavigation();
-  const [showPopupBlock, setShowPopupBlock] = React.useState(false);
-  const [channelInfo, setChannelInfo] = React.useState(null);
+  } = useGroupInfo(selectedChannel?.id);
+
   const {signedProfileId, anonProfileId} = useUserAuthHook();
+  const {localDb} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useGroupInfo` hook is now being called with `selectedChannel?.id` as a parameter. This change could potentially introduce null or undefined values if `selectedChannel` is not defined. Please ensure that the `useGroupInfo` hook can handle such cases gracefully.
```diff
-   } = useGroupInfo(selectedChannel?.id);
+   } = useGroupInfo(selectedChannel ? selectedChannel.id : defaultChannelId);
```
In this suggested change, `defaultChannelId` should be a valid default value for the channel id.


<!-- This is an auto-generated comment by OSS CodeRabbit -->